### ISSUE TITLE
Adopt OCI as the packaging and distribution format for Meshery Applications, Designs, Filters

### ISSUE THREAD

**Issue by leecalcote** [meshery/meshery#6447]
#### Mockups

---
#### Contributor [Guides](https://docs.meshery.io/project/contributing) and [Handbook](https://layer5.io/community/handbook)
- 🛠 [Meshery Build & Release Strategy](https://docs.meshery.io/project/build-and-release)
- 📚 [Instructions for contributing to documentation](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#documentation-contribution-flow)
   - Meshery documentation [site](https://docs.meshery.io/) and [source](https://github.com/meshery/meshery/tree/master/docs)
- 🎨 Wireframes and designs for Meshery UI in [Figma](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI)
- 🙋🏾🙋🏼 Questions: [Discussion Forum](https://discuss.layer5.io) and [Community Slack](http://slack.layer5.io)

**Comment by chethan27** [comment]
hi @leecalcote , can I work on this feature? if it is in your spring pipeline.

**Comment by leecalcote** [comment]
@chethan27 thanks for asking. I'd like for you to do, assuming that this is within your wheelhouse. Given the large nature of this feature, let's start with a design spec. First, it'd be good to get your other assigned issue completed, and then for you to see the community's shared drive for the [process on creating a functional design spec](https://docs.google.com/document/d/1RP3IWLc-MiQS-QYasqCoVuCH7--G87p5ezE5f_nOzB8/edit). 👍

**Comment by mak2002** [comment]
Hey @leecalcote, 

I am really excited to work on this project through LFX Mentorship. I am very excited about cloud native tech and have been a Google Summer of Code 2022 contributor.  

I was thinking of building an OCI Image Quality Analyzer. This would involve developing a tool to validate OCI images against the OCI specification. This will help me enhance my knowledge of Golang and OCI Images. However, I'd love to get your input on whether this aligns with the program's goals. If you have any other suggestions, I'm open to exploring them too.

**Comment by prateek041** [comment]
Hello @leecalcote I see that this issue is up in LFX mentorship this June session. I am quite interested in working on this since I recently learnt the skills necessary for this project (Graphql, React etc.) and this seems to be a perfect opportunity to polish them by working on it.

I am here to show my interests, would you suggest me what should I do next ?

**Comment by Meyazhagan** [comment]
Hi @leecalcote I went through how kyverno done a poc on storing policies in oci register and how OCI artifacts works. 
I would like to work on this feature. Is design spec is started or where can find more details on implementing?

**Comment by leecalcote** [comment]
@Meyazhagan, no, a new design spec needs to be created. You're empowered to do so. See [the instructions](https://github.com/meshery/meshery/issues/6447#issuecomment-1294057204) above in order to do so.

**Comment by Meyazhagan** [comment]
@leecalcote I guess i don't have access to 'make a copy' or move files. I already filled 'community member form'.

**Comment by satyampsoni** [comment]
What is the deadline to apply for this project for LFX Mentorship?

**Comment by Meyazhagan** [comment]
> import/export MeshModels as OCI images.
@leecalcote here MeshModel refers to combining all three that is application, design, filter (all in one OCI image) or referring to each separately

**Comment by leecalcote** [comment]
@Meyazhagan, separately.

**Comment by leecalcote** [comment]
Potentially fixed by https://github.com/meshery/meshery/pull/9807.

**Comment by sandramsc** [comment]
> What is the deadline to apply for this project for LFX Mentorship?

Hi @satyampsoni the question above is best posted on the Slack or [discussion forum](https://discuss.meshery.io/) as it does not contribute to resolving the specific issue at hand.

**Comment by leecalcote** [comment]
Design spec draft - https://docs.google.com/document/d/1AmNnwtMbYSVQ00TZRYtEZBJQTrPtH_S51ZG2UiwBGqs/edit
